tests/features/test_audio.py

```diff
@@ -0,0 +1,76 @@
+import sys
```

Comment:
Can we also add one test where we load a 44.1 kHz file with 16 kHz sampling rate? This means that `self.sampling_rate` has to be adapted before decoding the samples. This is the most common use case actually for all speech scripts.

Reply:
Done.